extensions/wikia/CommunityMessages/CommunityMessages.class.php

```diff
@@ -37,26 +37,12 @@ static function onSkinTemplatePageBeforeUserMsg(&$msgs) {
 		}
 
 		//get timestamp of message
-		$communityMessagesTimestamp = $wgMemc->get(wfMemcKey('CommunityMessagesTimestamp'));
-
-		if (!$communityMessagesTimestamp) {
-			$msgTitle = Title::newFromText('community-corner', NS_MEDIAWIKI);
-			if ($msgTitle) {
-				$msgRev = Revision::newFromTitle($msgTitle);
-				if ($msgRev) {
-					$communityMessagesTimestamp = wfTimestamp(TS_UNIX, $msgRev->getTimestamp());
-					$wgMemc->set(wfMemcKey('CommunityMessagesTimestamp'), $communityMessagesTimestamp, 86400 /*24h*/);
-				}
-			}
-		}
+		$cacheKey = static::getCommunityMessagesCacheKey();
+		$communityMessagesTimestamp = $wgMemc->get( $cacheKey );
```

Comment:
In below logic we already return if the message timestamp is older than 24 hours so I think this is safe this way